src/functions-and-arrays.js

```diff
@@ -1,43 +1,127 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
 
+function maxOfTwoNumbers(firstNumber, secondNumber) {
+  if (firstNumber > secondNumber) {
+    return firstNumber;
+  } else if (firstNumber < secondNumber) {
+    return secondNumber;
+  } else {
+    return firstNumber;
+  }
+}
```

Comment:
Esto esta perfect!!! pero una cosilla! si hacemos esto nos ahorramos una pregunta!
```suggestion
function maxOfTwoNumbers(firstNumber, secondNumber) {
  if (firstNumber < secondNumber) {
    return secondNumber;
  } else {
    return firstNumber;
  }
}
```